common/src/main/java/com/microsoft/identity/common/internal/util/ProcessUtil.java

```diff
@@ -72,4 +74,15 @@ private static String getProcessName(@NonNull final Context context) {
 
         return null;
     }
+
+    /**
+     * Returns a handler based on the current looper.
+     */
+    public static @NonNull Handler getPreferredHandler() {
+        if (null != Looper.myLooper() && Looper.getMainLooper() != Looper.myLooper()) {
```

Comment:
I'm not sure what the difference would be if we just said if (null != Looper.myLooper()) ?